--- bindings/vala/Makefile.am.bak	2009-03-04 12:49:45.000000000 +0100
+++ bindings/vala/Makefile.am	2009-08-09 19:19:58.000000000 +0200
@@ -5,7 +5,7 @@ DEPS_FILES = gst-rtsp-server-0.10.vapi
 gst-rtsp-server-0.10.deps:
 	cp packages/gst-rtsp-server-0.10.deps $@
     
-vapidir = $(VAPIDIR)
+vapidir = $(datadir)/vala/vapi
 vapi_DATA = $(VAPI_FILES) $(DEPS_FILES)
 
 EXTRA_DIST = packages $(DEPS_FILES)
